### ISSUE TITLE
Trouble finding Iron ore

### ISSUE THREAD

**Issue by Vasa200** [NillerMedDild/Enigmatica5#518]
**Modpack version**
Enigmatica5 0.5.5

**Issue**
Trouble finding iron ore. 

**Logs**
Input pastebin link to crash-report or latest log, if applicable

**Comment by NielsPilgaard** [comment]
Iron Ore generation is exactly the same as in vanilla Minecraft on Enigmatica 5 v0.5.5.

If you found 40 diamonds it sounds like you're mining too deep, try to mine around Y40

I understand that you're frustrated, but please keep it civil.

**Comment by Vasa200** [comment]
As frustrated as I am, this grumpy old man thought he was civil.  No iron digging down.  In fact  not much of any ore.  A couple of osmium in one group, , a couple of nickel in another.  Just gold, redstone, and diamonds.  I also spent many wasted hours on Tetra, so convoluted with tools for tools for tools.  I watched Mischief and he made it look easy, but I had to give up on it trying it myself after a few hours of dead ends.  It didn’t help I couldn’t find any iron.  JEI shows level zero up to 60 something.  That’s way OFF.   I didn’t want to make a bunch of tools just to fix and level other tools that needed more tools.  Nice concept, but in practice I found it GRINDY and a waste of effort and storage space.  I had to disable Yungs caves, the generation was horrible and convolute with one high tunnels and the like.  The surface was just as bad, all torn up and village after village was messed up with holes and ruined surface.  So far there are two mods that are definitely going bye bye and if I have to restart with another world again I’m going to freak.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by NielsPilgaard** [comment]
> Iron Ore generation is exactly the same as in vanilla Minecraft on Enigmatica 5 v0.5.5.
I remembered wrong, I did change pretty much all ore generation ^^'

Do you have the folder `openloader` in your modpack folder?
How did you install the modpack, and which worldgen did you pick?

**Comment by Vasa200** [comment]
Tetra is leaving my download of the mod pack.  I can’t mine iron with a stone pick axe.   I did branch off at 40 and found some iron… yeah, well, you see what I ran into with Tetra.  I don’t want to get a degree in convoluted logic to use a tool mod.    In vanilla, digging as many branches of several hundred meters each I would have found plenty of iron.  Talk about bucking the odds.  Disabling Tetra and trying again, well see how this goes.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by NielsPilgaard** [comment]
Vanilla tools still work just fine, nothing forces you to use Tetra, I don't really understand what the issue is.

**Comment by Vasa200** [comment]
I was just explaining where my frustration was coming from.  Didn’t mean to criticize the hard work put into this mod.  Must be just a fluke generation that I’ve experienced this time around.  So little iron with all the digging were beyond the odds, so just starting up Mek 3x to work past that.  Just watched Kryllyk’s video where he had his uber tool cursed beyond description trying to use Tetra, so I had to laugh it off…. Especially when he dumped the tool onto a “complaining” colonist.  Hmmmm.  Please don’t give me any cursed tools over this.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by Vasa200** [comment]
Issue probably boiled down to unusual world gen.  Found lots of diamonds and gold after digging but no iron.   I had to dig in another area in the other direction.  Tetra had me miffed as it wasn't working as expected.  When I finally did find iron I couldn't mine it despite leveling the tool.  Dumped the Tetra tools and went stone.  That was a lot of time spent on Tetra with the result unable to mine any ore.
